### ISSUE TITLE
Can no longer commit due to precommit - unhelpful error message

### ISSUE THREAD

**Issue by MatthewDaggitt** [vehicle-lang/vehicle#208]
I can no longer commit to the project as the precommit check fails with:
```
prettier.................................................................Failed
- hook id: prettier
- exit code: 1

prettier requires at least version 14 of Node, please upgrade
```

This is a very unhelpful error message as I have no idea what program `Node` actually is. I have no program called `Node` installed on my machine. I'm guessing its some variant of `npm` but that's at the latest version...

In general I'm worried about how difficult the dev environment is going to be to setup. The instructions in `HACKING.md` are going to have to be _very_ good...

**Comment by wenkokke** [comment]
Strange, prettier shouldn't require Node.

**Comment by wenkokke** [comment]
Options:
- remove prettier, which means no formatting for, e.g., json files, which we can probably live with;
- recommending folks install node (`npm` is to `node` as `cabal` is to `ghc`);
- you set `SKIP=prettier`.

**Comment by wenkokke** [comment]
Workaround is `SKIP=prettier git commit -m "<message>"` or `git commit -m "<message>" --no-verify`.

**Comment by MatthewDaggitt** [comment]
> remove prettier, which means no formatting for, e.g., json files, which we can probably live with;

I definitely vote for this one! We don't _want_ formatting for JSON files. All the JSON files are generated automatically from the Vehicle compiler. We shouldn't be formatting them....

**Comment by wenkokke** [comment]
It's turned off for .golden files anyway, but it's helpful for other JSON files.

**Comment by MatthewDaggitt** [comment]
What other JSON files do we have?

**Comment by wenkokke** [comment]
`test.json` files ^__^

**Comment by MatthewDaggitt** [comment]
No, it's really really not worth depending on `npm` just for that in my opinion.

**Comment by MatthewDaggitt** [comment]
Closed with [38aaf17](https://github.com/vehicle-lang/vehicle/commit/38aaf17df700a72b9e800b9fd9b78e82a33d90dd)

**Comment by MatthewDaggitt** [comment]
Fixed in 2c04902fad9b87b3bba8305dfbb13126c3b41f12
